### ISSUE TITLE
After commenting on a post the comment line does not erase itself

### ISSUE THREAD

**Issue by ankan1811** [mukeshphulwani66/Instagram-clone-MERN-Stack#15]
![issue1](https://user-images.githubusercontent.com/65054543/106351524-d2a44f00-6302-11eb-8f15-12be541b1b84.jpg)

Check this issue.
The comment line must be empty after commenting.

**Comment by moamlrh** [comment]
` 
const onSubmit = () => {
     setCommentValue('')
}
`

**Comment by Hartaj-Singh-Dev** [comment]
yes You have to create state  and then fill the input of comment box as <input value={comment} onChange={(event)=>{setcomment(event.target.value)}/>  and then you have to create a submit button like this <button onClick={()=>{Submit()}}>Post</button>
and now you can define the 'Submit()' function  as
const Submit = ()=>{
setcomment('')
}

**Comment by savarsri** [comment]
Fixed in pull request 46
